third_party/skia/src/core/SkMatrix.cpp

```diff
@@ -5,6 +5,7 @@
  * found in the LICENSE file.
  */
 
+#include "base/logging.h"
```

Comment:
This include should not be needed now.

Reply:
All changes in the //third_party were reverted